dap-mode.el

```diff
@@ -1138,18 +1155,20 @@ CONFIGURATION-SETTINGS - plist containing the preset settings for the configurat
         (file-error (setq success t))))
     port))
 
-(defun dap--select-template ()
-  "Select the configuration to launch."
+(defun dap--select-template (&optional origin)
+  "Select the configuration to launch.
+If origin is t, return the original configuration without prepopulation"
```

Comment:
Nit: We should capitalize "ORIGIN" in docs, following Emacs Lisp style.

Reply:
Done